documentation/aleo/04_opcodes.md

```diff
@@ -411,21 +429,215 @@ The instruction will halt if the given `String` or `Struct` value exceeds the 64
 
 | First     | Second   | Destination |
 |-----------|----------|:------------|
+| `Boolean` | `Scalar` | `Field`     |
+| `I8`      | `Scalar` | `Field`     |
+| `I16`     | `Scalar` | `Field`     |
+| `I32`     | `Scalar` | `Field`     |
+| `I64`     | `Scalar` | `Field`     |
+| `U8`      | `Scalar` | `Field`     |
+| `U16`     | `Scalar` | `Field`     |
+| `U32`     | `Scalar` | `Field`     |
+| `U64`     | `Scalar` | `Field`     |
+| `String`  | `Scalar` | `Field`     |
+| `Struct`  | `Scalar` | `Field`     |
+
+***
+
+### `commit.ped128`
+
+[Back to Top](#table-of-standard-opcodes)
+
+#### Description
+
+Computes a Pedersen commitment up to a 128-bit input in `first`, and some randomness in `second`, storing the commitment in `destination`. Randomness should always be a `Scalar` value, and the produced commitment will always be a `Group` value.
```

Comment:
"will always be a Group" --> "is a Field"

Reply:
For both `commit.ped64` and `commit.ped128`, the produced commitment is a `Field` value.

Reply:
Done.